charts/umbrella/README.md

```diff
@@ -255,6 +255,7 @@ Collection of hosts to be added to the `/etc/hosts` (Linux and Mac) or the `C:\W
 192.168.49.2    iatpmock.tx.test
 192.168.49.2    business-partners.tx.test
 192.168.49.2    pgadmin4.tx.test
+192.168.49.2    ssi-dim-wallet-stub.tx.test
```

Comment:
The data provider controlplane mapping entry is missing in hosts file.
`192.168.49.2    dataprovider-controlplane.tx.test`

Reply:
It is not in the umbrella, there is PR open about it: https://github.com/eclipse-tractusx/tractus-x-umbrella/pull/124

Reply:
Thank you for pointing this out! This issue has been addressed in PR #124 . When this PR is approved, I'll rebase the changes to ensure this update is included.